app/styles/components/meetings-schedule.scss

```diff
@@ -53,4 +110,160 @@
   tbody {
     cursor: pointer;
   }
-}
\ No newline at end of file
+}
+
+
+
+
+
+
+#schedule-header .form-control {
+	display: inline;
+	width: inherit;
+}
+
+
+
+/* filters */
+
+
+
+.filter-bar{
+}
+
+i{
+	margin-left: 8px;
+	margin-right: 8px;
+}
+
+.filter-dropdown{
+	color: #444;
+	transition: color 0.2s ease;
+	cursor: pointer;
+	margin-right: 16px;
+}
+
+.filter-dropdown:hover{
+	color: #666;
+}
+
+.filter{
+	padding: 8px 8px 8px 16px;
+	background-color: #3A3A3A;
+	border-radius: 50px 0 0 50px;
+	color: #e8e4e4;
+	margin: 8px 0;
+	transition: all 0.2s ease;
+}
+
+.filter-remove{
+	padding: 8px 16px 8px 8px;
+	background-color: #3A3A3A;
+	border-radius: 0 50px 50px 0;
+	color: rgba(255,255,255,0.8);
+	transition: all 0.2s ease;
+}
+
+.filter:hover, .filter-remove:hover{
+	color: white;
+	background-color: #585858;
+}
+
+.operator{
+	color: rgba(255,255,255,0.5);
+}
+
+.filter-remove{
+	color: rgba(255,255,255,0.5);
+	cursor: pointer;
+}
+
+.remove:hover{
+	color: white;
+
+}
+
+.edit-filter-modal{
+	background-color: #F2F3F5;
+	width: 260px;
+	position: absolute;
+	border-radius: 0 0 4px 4px;
+	box-shadow: 0 1px 3px 0 rgba(0,0,0,0.3);
+	transition: transform 0.1s ease, opacity 0.1s ease;
+	transform-origin: 0 0;
+	padding: 16px;
+	z-index: 10;
+}
+
+select{
+	width: 100%;
+	height: 32px;
+	background-color: white;
+	border: 1px solid rgba(0,0,0,0.125);
+	margin-bottom: 16px;
```

Comment:
these could interfere with regular styles. need to be namespaced; though these have only been moved from app.css- theyre styles that were added along with the filter bar for the schedule. Should leave for now - this pr is already big. 

In the future need to be careful when ^c ^v from the internet....